src/host/alias.cpp

```diff
@@ -244,7 +244,7 @@ std::unordered_map<std::wstring,
                                                     exeNameW));
 
         // Set the return size copied to the size given before we attempt to copy.
-        // Then multiply by sizeof(wchar_t) due to a long standing bug that we must preserve for compatibility.
+        // Then multiply by sizeof(wchar_t) due to a long-standing bug that we must preserve for compatibility.

```

Comment:
CJR has an article and notes that `longstanding` has not yet won: https://www.cjr.org/language_corner/long-hyphen.php 
But, in general, `long standing` isn't really preferred by anyone afaict...
https://www.merriam-webster.com/dictionary/long-standing